modules/i18n/src/translations/en-US/portals/applications.ts

```diff
@@ -2078,6 +2078,11 @@ export const applications: ApplicationsNS = {
                 },
                 attributeProfile: {
                     fields: {
+                        attributeNameFormat: {
+                            hint: "This specifies the format for attribute names in the attribute statement of the " +
```

Comment:
"specifies" sounds a bit formal / technical, IMO. how about this?

```suggestion
                            hint: "This defines the format for attribute names in the attribute statement of the " +
```

Reply:
This was the same pattern used in the hints for that form, which is why the same pattern was kept for the new field as well. Otherwise, it would be inconsistent. It's better to revisit all of this later.

<img width="1082" alt="Screenshot 2025-02-08 at 13 49 21" src="https://github.com/user-attachments/assets/5c5ffa34-29ac-4cf8-8df2-c3c063c797b1" />